tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -4,30 +4,31 @@
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import storage_cluster
-from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility.utils import ceph_health_check
 
 
 @ignore_leftovers
 @tier1
 @polarion_id('OCS-1191')
 class TestAddCapacity(ManageTest):
     """
+
     Automates adding variable capacity to the cluster while IOs running
     """
     def test_add_capacity(self):
         """
-        Test to add variable capacity to the OSD cluster while IOs running
-        """
-        self.ceph_cluster = CephCluster()
+       Test to add variable capacity to the OSD cluster while IOs running
+       """
+        assert ceph_health_check(), 'health check failed'
         osd_size = storage_cluster.get_osd_size()
         result = storage_cluster.add_capacity(osd_size)
         pod = OCP(
             kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
         )
         pod.wait_for_resource(
-            timeout=300,
+            timeout=1200,
             condition=constants.STATUS_RUNNING,
             selector='app=rook-ceph-osd',
             resource_count=result * 3
         )
-        self.ceph_cluster.cluster_health_check(timeout=1200)
+        assert ceph_health_check(), 'Test Failed, new pods failed reaching running state'
```

Comment:
Will this wait for 1200 seconds? or more?

Reply:
By default, timeout is 10* number of pods.